functions/wifi.bash

```diff
@@ -9,7 +9,24 @@ wifi_setup() {
       if ! (whiptail --title "No WiFi Hardware Detected" --yesno "$question" 10 80); then echo "FAILED"; return 1; fi
     fi
   fi
+
   if [ -n "$INTERACTIVE" ]; then
+    if grep -q "dtoverlay=disable-wifi" /boot/config.txt; then
+      if (whiptail --title "WiFi is currently disabled" --yesno "WiFi is currently disabled on your box. Enable ?" 7 55); then 
+        cond_echo "Removing 'dtoverlay=disable-wifi' from /boot/config.txt"
+        sed -i '/dtoverlay=disable-wifi/d' /boot/config.txt
```

Comment:
see above, ^ or something similar to ensure we do not remove comments...

Reply:
done

Reply:
> 
> 
> where?

L14

Reply:
I was looking at the sed command this time....